src/overrides.ts

```diff
@@ -50,7 +50,7 @@ class DefaultOverrides {
   readonly STARTING_BIOME_OVERRIDE: Biome = Biome.TOWN;
   readonly ARENA_TINT_OVERRIDE: TimeOfDay | null = null;
   /** Multiplies XP gained by this value including 0. Set to null to ignore the override */
-  readonly XP_MULTIPLIER_OVERRIDE: number | null = null;
+  readonly XP_MULTIPLIER_OVERRIDE: number | null = 50;
```

Comment:
```suggestion
  readonly XP_MULTIPLIER_OVERRIDE: number | null = null;
```
You're not supposed to use the overrides like this. :pikamad:

Reply:
sssh let me give my pokemon illegal exp boosts